Applications/SlicerApp/qSlicerAppMainWindow.h

```diff
@@ -33,29 +33,44 @@ class Q_SLICER_APP_EXPORT qSlicerAppMainWindow : public qSlicerMainWindow
 public:
   typedef qSlicerMainWindow Superclass;
 
-  qSlicerAppMainWindow(QWidget *parent=nullptr);
+  qSlicerAppMainWindow(QWidget * parent = nullptr);
   ~qSlicerAppMainWindow() override;
 
 public slots:
-  void on_HelpKeyboardShortcutsAction_triggered();
-  void on_HelpBrowseTutorialsAction_triggered();
-  void on_HelpDocumentationAction_triggered();
-  void on_HelpQuickStartAction_triggered();
-  void on_HelpGetHelpAction_triggered();
-  void on_HelpUserInterfaceAction_triggered();
-  void on_HelpVisitSlicerForumAction_triggered();
-  void on_HelpJoinUsOnLinkedInAction_triggered();
-  void on_HelpSearchFeatureRequestsAction_triggered();
-  void on_HelpViewLicenseAction_triggered();
-  void on_HelpHowToCiteAction_triggered();
-  void on_HelpSlicerPublicationsAction_triggered();
-  void on_HelpAcknowledgmentsAction_triggered();
-
-  void on_HelpReportBugOrFeatureRequestAction_triggered();
-  void on_HelpAboutSlicerAppAction_triggered();
+  void
```

Comment:
Keep single line. Splitting just adds noise.

Reply:
I'll change in next forced push.

Reply:
Done.

```
BreakAfterReturnType: Automatic
PenaltyReturnTypeOnItsOwnLine: 200 <- Default value high, greater than line length of 120, so not lilkely to split on return type.
```